src/Client.php

```diff
@@ -16,41 +16,64 @@
 
 class Client
 {
-    const THUMBNAIL_FORMAT_JPEG = 'jpeg';
-    const THUMBNAIL_FORMAT_PNG = 'png';
+    public const THUMBNAIL_FORMAT_JPEG = 'jpeg';
```

Comment:
Please don't add `public` here.

Reply:
Removed the access modifiers. I can open a new pull request for this later